bci_tester/data.py

```diff
@@ -622,6 +622,22 @@ def create_BCI(
     )
 ]
 
+
+POSTFIX_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/postfix:{postfix_ver}",
+        bci_type=ImageType.APPLICATION,
+        available_versions=os_versions,
+        forwarded_ports=[PortForwarding(container_port=25)],
+        extra_environment_variables={"SERVER_HOSTNAME": "localhost"},
+    )
+    for postfix_ver, os_versions in (
+        (3.7, ["15.5"]),
+        (3.8, ["15.6"]),
+        (3.9, ["tumbleweed"]),
```

Comment:
@dcermak, what's the general process of maintaining these hardcoded image tags?

Whenever the package versions are bumped in SLE SPs or TW, we bump these versions?

Reply:
> @dcermak, what's the general process of maintaining these hardcoded image tags?
> 
> Whenever the package versions are bumped in SLE SPs or TW, we bump these versions?

Yes, that's the procedure.